lesson32/src/js/register.js

```diff
@@ -1,22 +1,30 @@
 import { showErrorMessage,checkFormValidityInBlur, confirmIfCanSubmit } from "./modules/validation";
+import { toggleInertAttribute } from "./modules/toggle-inert";
 import { togglePasswordDisplay } from "./modules/togglepassword";
 
 const bodyElement = document.querySelector("body");
 const checkboxLink = document.getElementById("js-checkbox-link");
+const modal = document.getElementById("js-modal-area");
 const modalCloseButton = document.getElementById("js-modal-close-button");
 const modalInner = document.getElementById("js-modal-inner");
 const submitButton = document.querySelector(".js-submit-button");
 const invalidItems = document.getElementsByClassName("invalid");
+const focusControlTargets = [document.getElementById("js-form"), document.getElementById("js-header")];
 
 const openModal = () => {
     document.getElementById("js-modal-area").classList.add("is-active");
     bodyElement.classList.add("fixed");
     document.getElementById("js-modal-contents").scrollTop = 0;
+    toggleInertAttribute(focusControlTargets, true);
+    toggleInertAttribute([modal], false);
```

Comment:
And here we make it into an array.
```suggestion
    toggleInertAttribute([form, header], true);
    toggleInertAttribute([modal], false);
```

Reply:
@nakagawamai 
This is also corrected as well.
Thank you very much!
[d081c48](https://github.com/haru-programming/JavaScript-lessons/pull/60/commits/d081c48158664189805e51b4b38131c17fbda099)